functions.php

```diff
@@ -5,43 +5,37 @@ function Setupshop_script_enqueue() {
 
   }
   add_action('wp_enqueue_scripts', 'Setupshop_script_enqueue');
-
 ?>
 
 <?php
-
-  /**  ------- my additions below --------
-  * Register our sidebars and widgetized areas.
-  *
-  */
-  function arphabet_widgets_init() {
-
-	register_sidebar( array(
-		'name'          => 'Filters',
-		'id'            => 'filter',
-		'before_widget' => '<div>',
-		'after_widget'  => '</div>',
-		'before_title'  => '<h2 class="rounded">',
-		'after_title'   => '</h2>',
-	) );
-
-  }
+/**       ------- my additions below --------
+*         Register sidebars and widgetized areas.
+*
+*/
+function arphabet_widgets_init() {
+  register_sidebar( array(
+  	'name'          => 'Filters',
+  	'id'            => 'filter',
+  	'before_widget' => '<div>',
+  	'after_widget'  => '</div>',
+  	'before_title'  => '<h2 class="rounded">',
+  	'after_title'   => '</h2>', ) );
+}
   add_action( 'widgets_init', 'arphabet_widgets_init' );
```

Comment:
Consider your function names. Think about what it is you're copy pasting. The `arphabet` is nothing more than a prefix. You shouldn't just take that with. Just make sure the second part of the add_action is the function you're calling.